### ISSUE TITLE
TypeError: forward() got an unexpected keyword argument 'is_causal'

### ISSUE THREAD

**Issue by zqxyus** [gzerveas/mvts_transformer#68]

**Comment by zqxyus** [comment]
when i tried use mvts_transformer to SpokenArabicDigits ,  the error occurred

**Comment by annachester** [comment]
I have the same issue. Have you solved it yet?

**Comment by seay-12** [comment]
您好！我是一名计算机专业的学生，想要学习这份代码但是关于作者发布的关于分类的数据集已经无法打开，如果您方便的话可以加我的联系方式打包一份给我吗

**Comment by annachester** [comment]
I solved it by adding "**kwargs" to the forward() function in the TransformerBatchNormEncoderLayer class.. this solves the issue that the argument 'is_casual' is no longer expected in the function call.

**Comment by geetu040** [comment]
yes you can basically add `, is_causal: bool = None` or `, **kwargs` at the end of forward pass for `TransformerBatchNormEncoderLayer`

**Comment by valentin-fngr** [comment]
> I solved it by adding "**kwargs" to the forward() function in the TransformerBatchNormEncoderLayer class.. this solves the issue that the argument 'is_casual' is no longer expected in the function call.

this resolved the issue.

**Comment by Laplacat** [comment]
maybe you could use torch==1.7.1
